src/node/coinstats.h

```diff
@@ -29,7 +29,7 @@ enum class CoinStatsHashType {
 
 struct CCoinsStats {
     //! Which hash type to use
-    const CoinStatsHashType m_hash_type;
+    CoinStatsHashType m_hash_type;
```

Comment:
Wouldn't you have to delete the default constructor now? Otherwise it seems possible to leave this uninitialized?

Reply:
Hm, ok, didn't think about that. I opted to set hash_serialized as a default value instead.